Sources/Get/get().swift

```diff
@@ -17,9 +17,9 @@ import Utility
  - Throws: Error.InvalidDependencyGraph
  - Returns: The modules that this manifest requires building
 */
-public func get(manifest: Manifest) throws -> [Package] {
+public func get(manifest: Manifest, manifestParser: (path: String, url: String) throws -> Manifest) throws -> [Package] {
```

Comment:
I prefer not to typealias closures. It just means you have to hunt out the definition rather than have the information in front of you.

There is no potential badness that can happen from not typealiasing it that I can see, if I accidentally type the wrong signature the compiler will point it out.